yellowbrick/features/pcoords.py

```diff
@@ -371,8 +371,9 @@ def fit(self, X, y=None, **kwargs):
         if self.normalize is not None:
             X = self.NORMALIZERS[self.normalize].fit_transform(X)
 
-        # the super method calls draw and returns self
-        return super(ParallelCoordinates, self).fit(X, y, **kwargs)
```

Comment:
Although we're not returning `super` we probably do still want the call to super right?

Reply:
Call to super was basically a duplicate code. So I just removed it. It was used only to call draw.

Reply:
Ok, I see now that I've expanded that. In this case - though it is out of scope for this PR - we should have the super-classes handle this work and not ParallelCoordinates. Let's create an issue for this an handle it in a follow-up PR.